google/cloud/spanner/samples/samples.cc

```diff
@@ -564,7 +566,8 @@ void ReadWriteTransaction(google::cloud::spanner::Client client) {
     if (!read) {
       throw std::runtime_error(read.status().message());
     }
-    for (auto row : read->Rows<std::int64_t>()) {
+    using RowType = spanner::Row<std::int64_t>;
```

Comment:
nit: here (and a few places below) it seems like `read->Row<spanner::Row<std::int64_t>>()` would be a cleaner example.

Reply:
I wasn't sure which way looked the best. So I updated this (and the places below) to avoid the `RowType` alias if the alias was only used once, and the `Row` only contained one column.